### PR TITLE
add Photon browser load experiment

### PR DESCRIPTION
**Pull request by ericawright**
@jtg567 @phlsa @bwinton @rayborn 

This adds the Photon Loading experiement to the Braingames game family.

Note: 
 - ~~There were a lot of things that needed doing to get these experiments running that were not in the Readme. I'll make another PR to add more detailed instructions to the Readme~~. 
- ~~PG has been added in a separate file so we don't have to install it in dev~~

### CODE REVIEW

File: games/photon-loading/game.js [hunk 1]
```diff
@@ -0,0 +1,53 @@
+
```
Comment (bwinton) [inline games/photon-loading/game.js]: We should probably have a `"use strict";` here to get us into ES6/fast mode…

File: games/photon-loading/static/html/startup.html [hunk 2]
```diff
@@ -0,0 +1,126 @@
...
+          <div class="launch cohort-1" id="l2" title="Web Browser"></div>
+          <div class="launch cohort-2" id="l1" title="Web Browser"></div>
+      </div>
+
+      <button id="end-trial"></button>
+    <div/>
+
+    <script type="text/javascript">
+
+        function vars() {
```
Comment (bwinton) [inline games/photon-loading/static/html/startup.html]: Ya know, I bet we could pull this javascript out into its own file…

File: games/photon-loading/static/html/startup.html [hunk 3]
```diff
@@ -0,0 +1,126 @@
...
+<html lang="en">
+
+<head>
+    <title>Startup Concept</title>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+</head>
+<body>
+    <div id="active-game">
+      <span id="navigation">
```
Comment (bwinton) [inline games/photon-loading/static/html/startup.html]: Do we show this at all?  Should we just remove it?

Comment (ericawright) [inline games/photon-loading/static/html/startup.html]: oh, the navigation? nope not anymore

File: games/photon-loading/static/html/startup.html (hunk 2 above)
Comment (ericawright) [inline games/photon-loading/static/html/startup.html]: update - at first we thought we couldn't but then found a roundabout way with data - will do.

**Comment by phlsa** [comment]
A few copy changes/additions:

== Experiment Start Screen ==

**Thank you for participating!**

This web page will take you through a guided experiment. As part of doing that, it will simulate a desktop environment. You will automatically get back to your normal desktop environment at the end of the experiment.

The experiment will take you through the process of launching a web browser three times. It will also ask you some questions in between.

== If the user clicks the submit button without selecting one of the radio buttons (as an alert) ==

Please select a value

== End of experiment (currently there is only a blank screen here) ==

**Thank you again for participating**
You may now close this browser window.

**Comment by phlsa** [comment]
Also, could we add a unit to the selectable options (so 1s, 2s, 3s...)?

**Comment by ericawright** [comment]
@phlsa, all added in. Screenshots attached. jspsych gives us the "required" popup on the multi-choice question - I can change the styling if you really want me to.

screen shot 2017-04-03 at 3 36 38 pm
screen shot 2017-04-03 at 3 46 52 pm
screen shot 2017-04-03 at 3 51 14 pm

**Comment by phlsa** [comment]
Nice! 👍

**Comment by jtg567** [comment]
This looks ready to merge and test, I presume it works in local dev.

One thing - @glogiotatidis would you foresee the changes here to `tox.ini` and `requirements.txt` (plus `prod-requirements.txt`) causing problems? That's the only change to the platform here, I know it wouldn't affect my local dev env

**Comment by glogiotatidis** [comment]
> One thing - @glogiotatidis would you foresee the changes here to tox.ini and requirements.txt (plus prod-requirements.txt) causing problems? That's the only change to the platform here, I know it wouldn't affect my local dev env

Yes, heroku reads `requirements.txt` only and with the changes made here it'll not install the postgres library.

**Comment by ericawright** [comment]
@glogiotatidis okay. I'll change that part back. I suppose I can just use it that way for my local dev.

**Comment by bwinton** [comment]
As someone who might want to add an experiment in the future, and _really_ doesn't want to install Postgres, is there any way we could make it an optional dependency somehow?

**Comment by jtg567** [comment]
@bwinton it is only required in production - local dev looks for sqlite.

As a matter of fact, my local sqlite backend doesn't work, with the worst side effect being HTTP 500 errors when trying to POST to it. I check the DB actions on -stage.

**Comment by bwinton** [comment]
@jtg567 The problem we ran into was the pip install failing because it can't install psychopg because postgres isn't installed.  Running it would be fine, I suppose, but we couldn't get that far.

**Pull request merged**